composer.json

```diff
@@ -25,7 +27,10 @@
         "ext-mbstring": "*",
         "ext-curl": "*",
         "composer/ca-bundle": "~1.0",
-        "symfony/options-resolver": "~2.7|~3.0"
+        "symfony/options-resolver": "~2.7|~3.0",
+        "php-http/httplug": "~1.1",
+        "php-http/curl-client": "~1.7",
+        "zendframework/zend-diactoros": "~1.4"
```

Comment:
Yeah, put this in the require-dev instead, so you can run the tests. 

Also, do not forget to require `php-http/discovery`